components/runtime/erlang/Makefile

```diff
@@ -139,6 +139,10 @@ COMPONENT_POST_INSTALL_ACTION += \
 		$(PROTOUSRLIBDIR64)/erlang/bin/erl \
 		$(PROTOUSRLIBDIR64)/erlang/releases/RELEASES
 
+# hostname not FQDN must be resolvable or send_receive will fail
+COMPONENT_PRE_TEST_ACTION= /usr/sbin/ping -a `hostname`
```

Comment:
Please, avoid doing this, it's ugly...

Reply:
Better add comment. But in my case, fqdn is properly resolved

Reply:
In my case too. But the test uses erlNode which is the hostname not the FQDN.
It could be set with erl -sname FQDN.
But there is no way to reliable determine the FQDN.
Actually testing the most simple: localhost